### ISSUE TITLE
[Feature Request] 

### ISSUE THREAD

**Issue by THECorranHorn** [crowbartools/Firebot#1973]
**Describe the solution you'd like**
Custom Viewer Table Page Size quantity in Database Settings

**Additional context**
na

**Comment by CKY-** [comment]
is the 60 viewer per page Limit what you are referring too and making it load any obituary number that you type in?

**Comment by THECorranHorn** [comment]
> is the 60 viewer per page Limit what you are referring too and making it load any obituary number that you type in?

yep, would like a lot more customization to that menu, it's useful to not have to scroll through a million pages at times, there are a lot of examples of how websites and/or programs allow custom #'s in a field like this one

**Comment by Firebottle** [comment]
@THECorranHorn Could you give an example of why you'd need more than the 60? I feel like maybe there is a opportunity for us to improve something here beyond upping that page size.

Does the search not work find what you need? Does the column sorting need improved in some way?

Ultimately we want to make it easy to find the info you need in there other than having to scan through hundreds of rows of data.

**Comment by THECorranHorn** [comment]
@Firebottle I do always prefer to be able to just zoom way out and take in the full picture at times hence my request. However if that is decided to not be implemented perhaps some filtering functionality KIND of like excel or similar data tracking and processing applications? 

If that makes any sense.
